index.js

```diff
@@ -289,6 +295,8 @@ var UserAttributeHandler = require('../../../src/user-attribute-handler');
         this.onLogoutComplete = onLogoutComplete;
         this.onModifyComplete = onModifyComplete;
         this.setOptOut = setOptOut;
+
+        this.common = new Common();
```

Comment:
i don't think this is needed

Reply:
You're right. It was a relic from something I was experimenting with. I'm going to remove it.